app/javascript/tubes-to-rack/components/TubesToRack.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <lb-page>
+    <lb-loading-modal
+      v-if="loading"
+      :message="progressMessage"
+    />
+    <lb-main-content>
+      <b-card
+        bg-variant="dark"
+        text-variant="white"
+      >
+        <h3>Tube rack UI goes here</h3>
+        <p>
+          Scanned tubes will appear in a UI representation of the tube rack.
+          This will be similar in function to the plate UI but will be capable
+          of opening the tube information page when viewed after creation.
+        </p>
+      </b-card>
+    </lb-main-content>
+    <lb-sidebar>
+      <b-card
+        header="Scan tubes"
+        header-tag="h3"
+      >
+        <b-form-group
+          label="Scan the tube barcodes into the relevant rack coordinates:"
+          style="position:relative; height:460px; overflow-y:scroll;"
```

Comment:
Pop the style in the style section of the component rather than inline.

Also, not entirely sold on the fixed height, and it might be worth using a percentage, although I'm aware with CSS this is sometimes easier said than done when it comes to heights.

Reply:
I'll take a look at this, but these lines of code were copied verbatim from an existing Vue component so I'll need to fix it there as well.

Reply:
I wasn't able to find a better solution for the height as it's quite heavily nested and the parent is relying on the child for height constraints.  I tried using viewport units but they didn't really work better than this fixed height does.